src/components/instance-configurator/InstanceConfigurator.jsx

```diff
@@ -15,6 +17,8 @@ const styles = () => ({
 class InstanceConfigurator extends React.Component {
     state = {
         tableKey: Math.random(),
+        dialogOpen: false,
+        toDelete: {},
```

Comment:
It's usually more declarative to use `null` instead of an empty object (and easier to work with the var, also)